src/render.tsx

```diff
@@ -39,11 +54,18 @@ export function renderInternal<T>(
 ) {
   const {
     wrapper: Wrapper,
+    concurrentRoot,
     detectHostComponentNames = true,
     unstable_validateStringsRenderedWithinText,
-    ...testRendererOptions
+    ...rest
   } = options || {};
 
+  const testRendererOptions: TestRendererOptions = {
+    ...rest,
+    // @ts-expect-error incomplete typing on RTR package
+    unstable_isConcurrent: concurrentRoot ?? getConfig().concurrentRoot,
```

Comment:
Just curious, with RTR being deprecated, do you plan to remove it?

https://github.com/facebook/react/tree/main/packages/react-test-renderer

Reply:
@slorber I've got a working prototype for alternative test renderer built using React Reconciler (the package that virtually all alt React renderers use) in #1669. It passes all of RNTL tests and would be a good replacement for RTR. I still need to modify it for React 19 (in #1690). While transition to the new render will happen, there is no immediate urgency to replace RTR, as it still works with React 19, although with deprecation warning (which is btw disabled for RN testing).

My plan is as follows:
- RNTL v13 (alpha version already published) will be based on RTR and support React 18 & 19. 
- RNTL v14 will be based on the new renderer and support probably React 19+

Reply:
I see thanks 👍